sources/ClangSharp/Generated/CXCompletionResult.cs

```diff
@@ -1,8 +1,11 @@
 namespace ClangSharp
 {
-    public partial struct CXCompletionResult
+    public unsafe partial struct CXCompletionResult
     {
+        [NativeTypeName("enum CXCursorKind")]
```

Comment:
Another example of where "normalizing" the native type name would allow us to drop the attribute.